### ISSUE TITLE
FE Display gets weird at high values

### ISSUE THREAD

**Issue by MuteTiefling** [desht/ModularRouters#153]
### Minecraft Version
1.16.5

### Forge Version
36.2.22

### Mod Version
modular-routers-1.16.5-7.5.1-69

### Describe your problem, including steps to reproduce it
Stepped up the FE storage and transfer rate and noticed that the display in the router gets a bit weird:

```
	fePerEnergyUpgrade = 2000000
	#FE transfer rate (FE/t) per Energy Upgrade
	#Range: > 1
	feXferPerEnergyUpgrade = 20000
	#Should block-breaking modules drop XP where appropriate? (ore mining etc.)
```
Seems like after 32 million FE, it just goes blank. 
https://streamable.com/z66swc

**Comment by desht** [comment]
Yep, I can see that.  Although it's just the graphical bar display, interestingly the tooltip is still correct.  So not a sync issues as I first suspected...

Update: yeah silly integer arithmetic overflow when calculating bar height, easy fix.

**Comment by desht** [comment]
Unfortunate timing given I only just did a 7.5.2 release on Monday... guessing you want these configs in place for E6E?

**Comment by MuteTiefling** [comment]
They're for e6e, yeah, but there's no rush. It works fine. Just a visual glitch as you say. And release is still no where close ^_^

**Comment by desht** [comment]
Fixed in 7.5.3 release
